forge-game/src/main/java/forge/game/spellability/SpellAbility.java

```diff
@@ -519,6 +519,10 @@ public void setManaCostBeingPaid(ManaCostBeingPaid costBeingPaid) {
     public boolean isAbility() { return true; }
     public boolean isActivatedAbility() { return false; }
 
+    public boolean isTurnFaceUp() {
```

Comment:
Hanmac's version in #4569 should overwrite this with the new stuff

Reply:
yeah, i think your stuff can be merged first without problems